src/data/events/talks/test1.md

```diff
@@ -11,23 +11,17 @@ speakers:
     img: "../../../images/JoseCarreiras.jpg"
     linkedin: https://www.linkedin.com/in/ricardonevesaraujo/
     website: https://davidsopas.com
+    github: https://www.github.com
     twitter: https://twitter.com/dsopas
 
-  - name: Hélder Gonçalves
-    occupations:
-      - what: senior software engineer
-        where: outsystems
-    bio: É um Engenheiro de Software na OutSystems, que busca excelência no seu trabalho. Com uma mente curiosa e um espírito inquisitivo desafia os seus colegas a percepcionar se estão a tomar as decisões mais acertadas. Actualmente, desenvolve funcionalidades colaborativas que visam aumentar a produtividade e desempenho de equipas de trabalho OutSystems.
-
-    img: "../../../images/DiogoMonteiro.jpg"
-
+n_day: 3
 day: 2019-10-29
 start_time: 15h00
 end_time: 16h00
 place: B032
-type: 0-talk
+type: Palestra
```

Comment:
What is the correct type here, `talk` or `Palestra` ? In the test2.md file the type is defined as `talk`

Reply:
With regards to this, just a heads up: the 0 in ```0-talk``` is no longer necessary. I introduced this because of the way I thought the program page would work (like last year) but since it's different, it makes no sense to keep this format.

Reply:
It is supposed to be Palestra or Workshop, so a person can identify the type of activity, since both pages are otherwise the same